src/widgets/modals/SearchForm.js

```diff
@@ -102,9 +102,11 @@ export const SearchFormComponent = ({
 
   const onError = useCallback(responseError => {
     setError(responseError);
+    resetData();
     resetQueryUrl();
   }, []);
 
+  const resetData = useCallback(() => setData(''), []);
   const resetError = useCallback(() => setError(''), []);
   const resetQueryUrl = useCallback(() => setQueryUrl(''), []);
```

Comment:
Revisiting this, seems a bit OTT :laughing:. Decided to maintain consistency for now, but I think these methods are pretty redundant (as @joshxg and/or @mark-prins probably already have said)!